docs/apache-airflow-providers-influxdb/operators/influxdb.rst

```diff
@@ -0,0 +1,23 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:InfluxDBOperator:
+
+InfluxDBOperator
+=================
+
+Not Implemented.
```

Comment:
Why do we need this if not implemented? I assume you added it because of the how-to-guide in the `provider.yaml` ? you can remove the `how-to-guide` from the yaml as there is no content here.

Reply:
There was a build docs error, if this wasnt added.

Reply:
@subkanthi Now that this is merged I think it a nice followup task would be to add `InfluxDBOperator`  around query function of the hook. It seems very simple and probably useful it will also help to add some content into this rst :)